src/importer/obscura.js

```diff
@@ -0,0 +1,17 @@
+const { scrapFrom } = require('../scraper/gdoc');
+const { launcher } = require('../utils');
+
+const scrap = scrapFrom(
+  '1nhTGlHevXa6adT_nUrr8aSwt0MevlqlRA3K4_AAr7nE',
+  {
+    name: 'Obscura',
+    instagram: 'https://www.instagram.com/obscuracaps/',
+  },
+  ['pop'],
```

Comment:
please delete this. I don't think it is necessary

Reply:
Deleted it